jumpscale/sals/zos/pools.py

```diff
@@ -44,8 +44,12 @@ def create(self, cu: int, su: int, farm: str, currencies: List[str] = None) -> P
 
         node_ids = []
         for node in self._nodes.iter(farm_id=farm_id):
+            if currencies == ["FreeTFT"] and not node.free_to_use:
+                continue
             node_ids.append(node.node_id)
         for gw in self._gateways.iter(farm_id=farm_id):
+            if currencies == ["FreeTFT"] and not gw.free_to_use:
```

Comment:
can we move this string to a constant?

Reply:
let's do that separately to reduce the technical debt we have